app/views/catalog/_form_basic_info.html.haml

```diff
@@ -127,3 +127,26 @@
                               "data-confirm" => _("Are you sure you want to remove this Reconfigure Entry Point?"),
                               :remote => true,
                               :title => _("Remove this Reconfigure Entry Point"))
+
+        .form-group
+          %label.col-md-2.control-label{:title => _("Retirement Entry Point (NameSpace/Class/Instance)")}
+            = _('Retirement Entry Point')
+            %br
+            = _('State Machine (NS/Cls/Inst)')
+          .col-md-8{:title => @edit[:new][:retire_fqname]}
```

Comment:
Comparing with reconfigure, should reconfigure section has something like `.col-md-8{:title => @edit[:new][:reconfigure_fqname]}`?